views/options.html

```diff
@@ -23,12 +23,12 @@ <h1>ZAP Browser Extension</h1>
         <td>As set in the ZAP API options</td>
       </tr>
       <tr>
-        <td><label>Enable</label></td>
+        <td><label>Enable </label></td>
         <td><input type="checkbox" id="zapenable"/></td>
       </tr>
-      <tr>
-        <td><label>Record</label></td>
-        <td><input type="checkbox" id="zaprecordingactive"/></td>
+      <tr >
```

Comment:
Revert?

Reply:
No, now we don't need it here. This was being used to start or stop the recording.

Reply:
I meant the space added to the tr tag.

Reply:
Bump, not essential but if it's not necessary.